setup.py

```diff
@@ -25,6 +32,55 @@ def find_version(*file_paths):
 VERSION = find_version('torchtext', '__init__.py')
 long_description = read('README.rst')
 
+
+def get_extensions():
```

Comment:
I assume this is copy paste from torchvision?

Reply:
Yes. I'm not an expert in Python so I copied and changed to fit.